listenbrainz/webserver/views/test/test_profile.py

```diff
@@ -94,6 +94,15 @@ def test_delete(self, mock_publish_data_to_queue):
         user = db_user.get(self.user['id'])
         self.assertIsNone(user)
 
+    
+    def test_delete_listens(self):
+        self.temporary_login(self.user['login_id'])
```

Comment:
let's test that the page redirects to login if the user isn't logged in also.

Reply:
I have added the test.